go/vt/wrangler/reparent.go

```diff
@@ -1085,3 +1088,23 @@ func (wr *Wrangler) TabletExternallyReparented(ctx context.Context, newMasterAli
 	}
 	return nil
 }
+
+func slaveWasRunning(stopSlaveStatus *replicationdatapb.StopSlaveStatus) bool {
+	if stopSlaveStatus == nil {
+		return false
+	}
+
+	return stopSlaveStatus.Before.SlaveIoRunning || stopSlaveStatus.Before.SlaveSqlRunning
```

Comment:
Continued my above comment on thread states; does `Connecting` count to be running? I think the answer is "it depends on why you're asking". The server is not effectively replicating right now, but _could be_ in a moment.

Reply:
As of now, we only check if == "Yes"

Reply:
Of course we can call is debt for now, but please be advised I've been bitten by this in production.

Reply:
What we want in this particular case is to ask, "did the instance _want_ to be replicating?" The enum we discussed elsewhere would let us answer this. Another thing to consider is maybe we should not be inferring "want" from MySQL-level status. Maybe we should instead be asking the tablet, "what was the value of `tm.slaveStopped()` before we told you to stop?"